index.js

```diff
@@ -1,19 +1,17 @@
 'use strict';
 const flags = require('flags');
 const fs = require('fs');
-const tmp = require('tmp');
 
-const extractScriptsFromHTML = require('./src/extractScripts.js');
+const {extractScriptsFromHTML} = require('./src/extractScripts.js');
```

Comment:
super nit: the temptation might be to reorder these alphabetically even though you have them listed in order of processing steps - maybe a brief comment above, like `The order of operation is to extract js from HTML, transform it, then inject it back into HTML` might be handy